### PR TITLE
Swarm creation script for development

### LINKED ISSUE #80: Simple swarm creation script
There is a need for a simple script that will create a simple swarm for development purposes.

### PR DESCRIPTION
**Pull request by devosray**
This feature introduces two new scripts in the `demo-swarm` folder:
```
start-demo-swarm.sh
remove-demo-swarm.sh
```

They can be used to create a very simple swarm for development purposes. As of now, the scripts create 2 NGINX stacks. The stacks serve simple HTML files on ports `8001` and `8002`.

The scripts don't start the Docks-API or Docks-UI containers; you have to start them yourself. 

After `start-demo-swarm.sh` is executed, the current machine will be part of and the only node in the swarm with 2 stacks on it. The Docks-API can be called on swarm-specific endpoints now, for example:

```
curl  http://localhost:8080/docker/services
-> [
  {
    "ID": "pv89iq5plzlfjffm2x77oa4ys",
    "Version": {
      "Index": 15
    },
    "CreatedAt": "2018-05-09T07:00:02.090763673Z",
    "UpdatedAt": "2018-05-09T07:00:02.104893636Z",
    "Spec": {
      "Name": "demo-nginx-1_web",
      "Labels": {
        "com.docker.stack.image": "nginx",
        "com.docker.stack.namespace": "demo-nginx-1"
      }, ...
```

This closes #80

### CODE REVIEW

**Comment by devosray** [comment]
@FJMentz I would like you to test it out on OSX, please.

File: demo-swarm/start-demo-swarm.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+echo "----- Trying to init swarm... (will fail if already part of one)"
+sudo docker swarm init
+
+#echo "----- Creating registry for images"
```
Comment (devosray) [inline demo-swarm/start-demo-swarm.sh:6]: I'm leaving the registry creating scripts as comments since we will probably use that in the future

**Review by FJMentz — COMMENTED** [review]
Well... So far it broke my docker... Had to force quit... Still waiting for docker to start working again... been 10minutes...

**Comment by FJMentz** [comment]
@devosray Well... So far it broke my docker... Had to force quit... Still waiting for docker to start working again... been 10minutes...

**Review by CDuPlooy — APPROVED** [review]

**Comment by devosray** [comment]
@FJMentz I suggest you run the commands one by one after your reinstall. The commands in the script are all really simple and should not cause anything like that. See where it gets stuck when you run it manually.

**Review by egeldenhuys — APPROVED** [review]

**Comment by egeldenhuys** [comment]
Thank you for the nice pull request comment :+1:

**Review by FJMentz — APPROVED** [review]
After finally getting my laptop at a stage where I could test this, I can see that it is working perfectly ✨Thank you @devosray

**Pull request merged**